### ISSUE TITLE
[Bug]: YoutubeTranscriptReader has dependency on outdated youtube-transcript-api with bugged urllib3 version

### ISSUE THREAD

**Issue by TFielloii** [run-llama/llama_index#10765]
### Bug Description

The new YoutubeTranscriptReader from llama-index-integrations has a dependency on youtube-transcript-api version 0.5.0 which at the time used a version of urllib3 that was notorious for truncating large compressed text files, the exact kind used by Youtube's transcript api. It's worth noting that before the newest integration change, I was using the latest youtube-transcript-api version (0.6.2) with no issues.

tl;dr: The new YoutubeTranscriptReader integration will not work until the youtube-transcript-api version is bumped up.

### Version

0.10.3

### Steps to Reproduce

pip install llama-index-readers-youtube-transcript

from llama-index-readers-youtube-transcript import YoutubeTranscriptReader

loader = YoutubeTranscriptReader()
documents = loader.load_data(
    ytlinks=["https://youtu.be/KRnDvNWXb20?si=G8EqJFtl2fomZvpT"]
)

**Comment by logan-markewich** [comment]
thats fair, should be easy enough to bump the dep vesrion

**Comment by eblis0n** [comment]
can not   use in window   Could not retrieve a transcript for the video https://www.youtube.com/watch?v=tgrkRVzNWc8! This is most likely caused by:

Subtitles are disabled for this video

If you are sure that the described cause is not responsible for this error and that a transcript should be retrievable, please create an issue at https://github.com/jdepoix/youtube-transcript-api/issues. Please add which version of youtube_transcript_api you are using and provide the information needed to replicate the error. Also make sure that there are no open issues which already describe your problem!
